cmd/bnscli/cmd_gov.go

```diff
@@ -0,0 +1,128 @@
+package main
```

Comment:
When building other cli tools with similar shape (eg. bnsd, gaiacli), I used a `commands` subdirectory with all the commands (and flag parsing). This did allow some (minimal) testing of that code.

The `main` package would just be one file that would import these commands and grab the first argument off the cli and dispatch down to the proper command.

I found cobra/viper to provide quite a few helpers, but also be heavy, opinionated libraries. If you wish to pull in some libraries beyond stdlib to make cli dev easier, I'm very open to that

Reply:
If they are really generic, they could go into `commands/client`, next to `commands/server`.
If they are bnscli specific, then `cmd/bnscli/commands` is a good place

Reply:
Can you explain a bit more what is the advantage of using a subdirectory? The current implementation is all in `main` package and I am able to test it.
None of the functionality is generic. They rely heavily on using a command line interface and they are complete in their functionality. I do not think we will need this code anywhere else.

I was wondering about using a command line library. As always I am very cautious of adding a dependency. I hope I can implement all I need without importing big dependencies. Only if the advantage is huge I will switch :smirk: